gateway/uplink.go

```diff
@@ -181,7 +183,7 @@ func executeDecoder(ctx context.Context, script string, vars map[string]interfac
 	}
 
 	resultChan := make(chan result)
-	timeoutCtx, _ := context.WithTimeout(ctx, 10*time.Millisecond)
+	timeoutCtx, cancel := context.WithTimeout(ctx, 10*time.Millisecond)
```

Comment:
let's make sure this new behaviour works as expected.

Reply:
tested that module still works correctly with all these changes!